### PR TITLE
Add Tar to the Herblore Skill calculator

### LINKED ISSUE #3504: Add Tar to the Herblore Skill calculator
While usually used in tick manipulations methods, making Guam, Marrentill, Tarromin, and Harralander tar is the best experience that can be gotten out of these low level herbs (except for Harralander), at the cost of using 15 tar per herb. Being able to use them in the calculator would be very useful.

### PR DESCRIPTION
**Pull request by rbbi**
Closes #3504

### CODE REVIEW

File: runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_herblore.json [hunk 1]
```diff
@@ -47,6 +47,12 @@
       "icon": 3410,
       "name": "Serum 207 (3)",
       "xp": 50
+    },
+	{
```
Comment (Adam-) [inline runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_herblore.json]: Format this consistently with the rest of the file.

Comment (rbbi) [reply runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_herblore.json]: @Adam- I believe this is just the diff showing the indenting incorrectly. I have looked at the file in its current state and looks fine? Unless i'm missing something...

Comment (Abextm) [reply runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_herblore.json]: Its a tabs/spaces thing

Comment (rbbi) [reply runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_herblore.json]: Done

**Comment by rbbi** [comment]
@deathbeam any update on this?

**Comment by deathbeam** [comment]
travis check is failing on this PR looks like

**Pull request merged**